packages/editor-preview/src/browser/editor-preview-widget-factory.ts

```diff
@@ -0,0 +1,45 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import URI from '@theia/core/lib/common/uri';
+import { EditorWidgetFactory } from '@theia/editor/lib/browser/editor-widget-factory';
+import { injectable } from '@theia/core/shared/inversify';
+import { EditorPreviewWidget } from './editor-preview-widget';
+import { NavigatableWidgetOptions } from '@theia/core/lib/browser';
+
+export interface EditorPreviewOptions extends NavigatableWidgetOptions {
```

Comment:
Do we need both interfaces?
https://github.com/eclipse-theia/theia/pull/9518/files#diff-075aefa08a4fb3b0224a5bcc327bf6ed6f9b9aa1d8ff0fa9ad159b1a89406b3cR27

Reply:
Likely not. Kept the one in the factor, removed it here. 😅